From 66a83c97c4545ce4f07e0d90998f906fae49caf2 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Tue, 22 Jul 2008 13:16:29 +0100
Subject: [PATCH] fix-glamo-mci-set-default-drive-level-0.patch

Tests on access to SD Card with Glamo drive level "0" show
that it reduces SD_CLK energy at 1.5GHz by 24dBm compared to
drive level 3.  This puts it only 6dB above the background
noise floor compared to 30dB and should make a solution for
GPS trouble with SD Card in.

SD card communication seems unaffected so far on the Sandisk
512MB card we ship.

Signed-off-by: Andy Green <andy@openmoko.com>
---
 drivers/mfd/glamo/glamo-mci.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mfd/glamo/glamo-mci.c b/drivers/mfd/glamo/glamo-mci.c
index 6585821..45b0afc 100644
--- a/drivers/mfd/glamo/glamo-mci.c
+++ b/drivers/mfd/glamo/glamo-mci.c
@@ -66,7 +66,7 @@ module_param(sd_max_clk, int, 0644);
  * for example
  */
 
-static int sd_drive = 2;
+static int sd_drive;
 module_param(sd_drive, int, 0644);
 
 /*
-- 
1.5.6.5

